engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-repositories.controller.js

```diff
@@ -49,6 +49,7 @@ angular.module('Bastion.products').controller('ProductRepositoriesController',
         $scope.successMessages = [];
         $scope.errorMessages = [];
 
+        $scope.downloadPolicies = [{id: 'immediate', name: 'Immediate'}, {id: 'on_demand', name: 'On Demand'}, {id: 'background', name: 'Background'}];
```

Comment:
In the case of checksum in the line below this, the names are translated where appropriate. What I am suggesting is that https://github.com/Katello/katello/pull/5758/files#diff-d985390622f59b70e39a9776ef0b59b9R157 duplicates knowledge with this structure that could be centralized in an angular service.